### DISCUSSION TITLE
the longhorn can not work on kubernetes v1.23

### DISCUSSION THREAD

**Discussion by rebotrer** [longhorn/longhorn#3962] [General]
In the kubernets v1.23 , the CRD API  ( apiextensions.k8s.io/v1beta1) is no longer used . So after I Installed longhorn . the pod 
longhorn-manager-t6fsh can not work normally and info that "  
E0510 14:33:24.989602       1 reflector.go:123] k8s.io/client-go/informers/factory.go:134: Failed to list *v1beta1.CSIDriver: the server could not find the requested resource
E0510 14:33:25.991338       1 reflector.go:123] k8s.io/client-go/informers/factory.go:134: Failed to list *v1beta1.CSIDriver: the server could not find the requested resource
"

**Comment by innobead** [comment]
What version u used? This has been supported from Longhorn 1.2, https://github.com/longhorn/longhorn/releases/tag/v1.2.0. You can probably use the latest 1.2.4 to try. 

However, 1.1 does not support K8s 1.22+ because of https://github.com/longhorn/longhorn/issues/2971#issuecomment-972669208
